From b15ebd5b28af79e45661b8c7da7a578528349e3d Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sat, 14 Mar 2020 07:26:37 +0200
Subject: [PATCH] [SW-0] goya2: preboot: set text base

It should be in ICCM and not FW_SRAM like ppboot

Change-Id: Iaac18b4bc47253096bd40b6068b86e4e79fff0bb
---

diff --git a/goya2_arc/common.h b/goya2_arc/common.h
index 7be24fb..4b42b08 100644
--- a/goya2_arc/common.h
+++ b/goya2_arc/common.h
@@ -8,7 +8,7 @@
 #include <preboot.h>
 
 #define PPBOOT_TEXT_BASE	0x1B05000C
-#define PREBOOT_TEXT_BASE	(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
+#define PREBOOT_TEXT_BASE	0xC
 
 #define mmCPU_PLL_BASE		0x4A2000
 #define mmIC_PLL_BASE		0x4A3000
